lpyramid.cpp

```diff
@@ -50,7 +52,7 @@ void LPyramid::convolve(std::vector<float> &a,
     assert(b.size() > 1);
 
     #pragma omp parallel for shared(a, b)
-    for (auto y = 0u; y < weight_; y++)
```

Comment:
I guess this is because https://stackoverflow.com/questions/2820621/why-arent-unsigned-openmp-index-variables-allowed in Visual Studio.